web/src/condense.ts

```diff
@@ -262,14 +281,22 @@ export function condense_and_collapse(elems) {
     }
 }
 
-export function initialize() {
+export function initialize(): void {
     $("#message_feed_container").on("click", ".message_expander", function (e) {
         // Expanding a message can mean either uncollapsing or
         // uncondensing it.
         const $row = $(this).closest(".message_row");
         const id = rows.id($row);
+        if (typeof id !== "number") {
```

Comment:
There is nothing wrong with id's return type (`number`) .I think using `typeof` for handling `undefined` degrades code readability as it confuses readers about `id`'s datatypes.

Reply:
@LaPulgaaa thanks for the review.